standard/conversions.md

```diff
@@ -25,17 +25,23 @@ Some conversions in the language are defined from expressions to types, others f
 > ```csharp
 > enum Color { Red, Blue, Green }
 >
-> // The expression 0 converts implicitly to enum types
-> Color c0 = 0;
+> class A
```

Comment:
The introduction of an extra class and method here still bothers me.

If we have a new template of "library code in a class" we could stick with the original code - the statements would be introducing instance fields, and the enum would be nested. A bit weird, but legal.

Any thoughts on that?

Reply:
Will try this in a new commit...

Reply:
Yes, this works. I'm not sure it's going to be a great long-term solution, but I think we'll want the new template anyway.